Dockerfile

```diff
@@ -1,38 +1,33 @@
-FROM node:10.15-stretch
-
-MAINTAINER datapunt.ois@amsterdam.nl
+FROM node:10.15-stretch  AS builder
+LABEL maintainer="datapunt@amsterdam.nl"
 
 EXPOSE 80
 
-RUN apt-get update && \
-  apt-get upgrade -y --no-install-recommends && \
-  apt-get install -y git nginx \
-    gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 \
-    libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 \
-    libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 \
-    libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation \
-    libappindicator1 libnss3 lsb-release xdg-utils \
-    xvfb libgtk2.0-0 libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 && \
-  rm -rf /var/lib/apt/lists/*
+WORKDIR /app
+
+COPY package.json \
+  package-lock.json \
+  .eslintrc.js \
+  .gitignore \
+  /app/
 
-COPY package.json /app/
-# RUN rm /etc/nginx/sites-enabled/default
+#  Changing git URL because network is blocking git protocol...
+RUN git config --global url."https://".insteadOf git://
+RUN git config --global url."https://github.com/".insteadOf git@github.com:
 
-WORKDIR /app
 
-ENV PATH=./node_modules/.bin/:~/node_modules/.bin/:$PATH
-RUN git config --global url."https://".insteadOf git:// && \
-    git config --global url."https://github.com/".insteadOf git@github.com: && \
-    npm --production=false --unsafe-perm install && \
-    chmod -R u+x node_modules/.bin/
+# Install NPM dependencies.
+RUN npm --production=false --unsafe-perm install && \
```

Comment:
Wouldn't `npm ci` be enough? `npm install` tends to alter the `package.json` which might include unwanted versions of packages